blocks/form/form.js

```diff
@@ -56,8 +56,16 @@ function createButton(field) {
       if (checkValidity(form) !== null) {
         button.setAttribute('disabled', '');
         await submitForm(form);
-        const redirectTo = field.Extra;
-        window.location.href = redirectTo;
+        const formContainer = form.closest('.form-container');
+        formContainer.firstElementChild.classList.add('hide');
+        form.classList.add('hide');
+        const paragraph = document.createElement('p');
+        paragraph.classList.add('contact-us');
```

Comment:
This class seems like it would be related to a contact-us form - is it the right class to use?

Reply:
I could change the name of the class.